frontend/home/home-card.component.tsx

```diff
@@ -87,4 +87,5 @@ type HomeCardProps = {
   title: string;
   iconUrl: string;
   link: string;
+  alt: string;
```

Comment:
I added this prop for accessibility and then added the prop to all places that use the HomeCard component